toolchain/check/convert.cpp

```diff
@@ -622,6 +680,19 @@ static auto PerformBuiltinConversion(Context& context, Parse::Node parse_node,
     }
   }
 
+  // A struct {.f_1: T_1, .f_2: T_2, ..., .f_n: T_n} converts to a class type
+  // if it converts to the struct type that is the class's representation type,
+  // that is, a struct with the same fields as the class, plus a base field
+  // where relevant.
```

Comment:
```suggestion
  // if it converts to the struct type that is the class's representation type
  // (a struct with the same fields as the class, plus a base field
  // where relevant).
```

Try not to have comma-separated parentheticals inside of comma-separated parentheticals.